metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -54,40 +56,54 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def safe_list_get(lst: List, idx: int) -> Any:
```

Comment:
recommendation:
1. since you're only ever using this to get the first element, calling it `unlist` and removing the idx argument may be more obvious.
2. same as the `single()` method used to do, perhaps its good to log a warning if >1 item exists

Reply:
We already had a safe map key get some I was trying to follow this, but I could also have it be `get_single_result` or something, the word `unlist` feels not descriptive enough to me